doc/faq.rst

```diff
@@ -122,3 +136,125 @@ file:
 
 The ``ocaml-print-intf`` program has special support for Dune, so it will
 automatically understand external dependencies.
+
+How Can I Build a Single Library?
+=================================
+
+You might want to do this when you don't have all the dependencies installed to
+compile an entire project, or parts of the project don't build for whatever
+reason. Maybe you want to check if your changes compile or produce build
+artifacts needed by ``ocaml-lsp-server``.
+
+Suppose you have a library defined in ``src/foo/dune``:
+
+.. code:: dune
+
+  (library
+   (public_name my_library)
+   ...)
+
+You can build this library on its own by running the following from the project
+root directory:
+
+.. code:: bash
+
+   $ dune build %{cmx:src/foo/my_library}
```

Comment:
Better is `%{cmxa:..}` or `%{cma:..}` to build the entire archive rather than the object for a specific module.

Reply:
changed to `cmxa`